server/rooms.ts

```diff
@@ -1709,6 +1755,8 @@ export const Rooms = {
 		let roomTitle;
 		if (gameType === 'multi') {
 			roomTitle = `Team ${p1name} vs. Team ${p2name}`;
+		} else if (options.title) {
```

Comment:
This should either be first or last in this if-else chain, right? I’m thinking first.